wix-formbuilder/$w/WixForms/guides/Migrate from WixForms.md

```diff
@@ -0,0 +1,38 @@
+# Migrate from WixForms to WixFormsV2
+As of April 2025, [WixForms](https://dev.wix.com/docs/velo/velo-only-apis/$w/wix-forms/introduction) is deprecated and any new forms created will only work with [WixFormsV2](https://dev.wix.com/docs/velo/velo-only-apis/$w/wix-forms-v2/introduction). You can access the functionality of WixForms using [WixFormsV2](https://dev.wix.com/docs/velo/velo-only-apis/$w/wix-forms-v2/introduction).
+
+Old forms will only work with the old APIs, WixForms, WixFormSubmitEvent, WixFormSubmittedErrorEvent, and WixFormSubmittedEvent. You can see which forms are old forms and which are new forms by looking at the forms and submissions page in a site dashboard, there's a tab for each. There is no easy way to distinguish in the editor, except that their menus are different.
+
+Below is a list of the deprecated WixForms components and functions and their alternatives in WixFormsV2.
+
+### [`collapsed`](https://dev.wix.com/docs/velo/velo-only-apis/$w/wix-forms/collapsed)
+Instead of `collapsed`, use [`collapsed`](https://dev.wix.com/docs/velo/velo-only-apis/$w/wix-forms-v2/collapsed) from WixFormsV2.
+
+### [`hidden`](https://dev.wix.com/docs/velo/velo-only-apis/$w/wix-forms/hidden)
+Instead of `collapsed`, use [`hidden`](https://dev.wix.com/docs/velo/velo-only-apis/$w/wix-forms-v2/hidden) from WixFormsV2.
+
+
+### [`isVisible`](https://dev.wix.com/docs/velo/velo-only-apis/$w/wix-forms/is-visible)
+Instead of `collapsed`, use [`isVisible`](https://dev.wix.com/docs/velo/velo-only-apis/$w/wix-forms-v2/is-visible) from WixFormsV2.
+
+### [`onWixFormSubmit()`](https://dev.wix.com/docs/velo/velo-only-apis/$w/wix-forms/on-wix-form-submit)
+Instead of adding `onWixFormSubmit()` to run when a site visitor starts to submit a Wix Form yet before the form is actually submitted and sent to the server, use [`onSubmit()`](https://dev.wix.com/docs/velo/velo-only-apis/$w/wix-forms-v2/on-submit) from WixFormsV2.
+
+### [`onWixFormSubmitted()`](https://dev.wix.com/docs/velo/velo-only-apis/$w/wix-forms/on-wix-form-submitted)
+Instead of adding `onWixFormSubmitted()` to run when a site visitor starts to submit a Wix Form yet before the form is actually submitted and sent to the server, use [`onSubmitSuccess()`](https://dev.wix.com/docs/velo/velo-only-apis/$w/wix-forms-v2/on-submit-success) from WixFormsV2.
```

Comment:
This description is incorrect - it seems you accidentally replicated the previous one.